src/redux/bidData/index.ts

```diff
@@ -89,7 +105,7 @@ export function BidReducer(state: BidState = defaultState, action: BidActionType
       return {
         ...state,
         isLoading: false,
-        bids: action.payload,
+        bidsBySaleId: { ...state.bidsBySaleId, ...action.payload },
```

Comment:
It's better that we don't move `bidsBySaleId` too much around. Narrow down the reducer.

Reply:
What do you mean by
 >Narrow down the reducer?

In this part, I was thinking of having the redux store each sequential bid and updating the store with new bids rather than requesting the entire history on each fetch.

Reply:
I've reworked this part now